cmd/todd-agent/main.go

```diff
@@ -51,6 +51,11 @@ func main() {
 
 	cfg := config.GetConfig(arg_config)
 
+	// Create optDir
+	if err := os.MkdirAll(cfg.LocalResources.OptDir, 0755); err != nil {
```

Comment:
One of the reasons I had the makefile create the optdir was so that if sudo was required, it was required there, where it was canonical (most folks are accustomed to running ```make install``` with sudo, I'd expect). 

Now, if the opt dir doesn't exist (which it won't unless you're in the provisioned vagrant machine), you're given a permissions error when this runs.